src/wordpress-recommendations.lib.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+/**
+ * Code related to the wprecommendations.lib.php checks.
+ *
+ * PHP version 5
+ *
+ * @category   Library
+ * @package    Sucuri
+ * @subpackage SucuriScanner
+ * @author     Northon Torga <northon.torga@sucuri.net>
+ * @copyright  2010-2019 Sucuri Inc.
+ * @license    https://www.gnu.org/licenses/gpl-2.0.txt GPL2
+ * @link       https://wordpress.org/plugins/sucuri-scanner
+ */
+
+if (!defined('SUCURISCAN_INIT') || SUCURISCAN_INIT !== true) {
+    if (!headers_sent()) {
+        /* Report invalid access if possible. */
+        header('HTTP/1.1 403 Forbidden');
+    }
+    exit(1);
+}
+
+/**
+ * Make sure the WordPress install follows security best practices.
+ *
+ * @category   Library
+ * @package    Sucuri
+ * @subpackage SucuriScanner
+ * @author     Northon Torga <northon.torga@sucuri.net>
+ * @copyright  2010-2019 Sucuri Inc.
+ * @license    https://www.gnu.org/licenses/gpl-2.0.txt GPL2
+ * @link       https://wordpress.org/plugins/sucuri-scanner
+ * @see        https://sitecheck.sucuri.net/
+ */
+class SucuriWordPressRecomendations
+{
+    
+    /**
+     * Generates the HTML section for the WordPress recommendations section.
+     *
+     * @return string HTML code to render the recommendations section.
+     */
+    public static function pageWordPressRecommendations()
+    {
+        
+        $params = array();
+        $recommendations = array();
+        $params['WordPress.Recommendations.Content'] = '';
+        
+        /**
+         * BEGIN security checks.
+         * 
+         * Each check must register a second array inside $recommendations,
+         * containing the title and description of the recommendation.
+         */
+
+        // Check if php version needs to be upgraded.
+        if (defined('PHP_MAJOR_VERSION') && PHP_MAJOR_VERSION <= 5) {
```

Comment:
In the hardening page we check `if (intval(version_compare(PHP_VERSION, '5.6.0') >= 0))`. I suggest removing this item from the hardening page (since we don't actually apply any hardening to the PHP version, we just check the version there), and checking here for PHP_VERSION below 7.1.0

In some cases older PHP might have backported security patches and we'll give a false warning, but that's ok.